deepchecks/nlp/utils/text_embeddings.py

```diff
@@ -30,6 +70,17 @@ def calculate_builtin_embeddings(text: np.array, model: str = 'miniLM',
         by using the command openai.api_key = YOUR_API_KEY
     file_path : Optional[str], default 'embeddings.csv'
         If given, the embeddings will be saved to the given file path.
+    device : str, default None
+        The device to use for the embeddings. If None, the default device will be used.
+    long_doc_averaging : str, default 'warn'
+        How to handle long documents (docments that are longer than the model context window).
+         Can be either 'average+warn', 'average', 'truncate' or 'raise'.
+         Currently, applies only to the 'open_ai' model, as the 'miniLM' model can handle long documents. Averaging is
+         done as described in https://github.com/openai/openai-cookbook/blob/main/examples/Embedding_long_inputs.ipynb
```

Comment:
same here but lets discuss in above thread

Reply:
Not sure what is same here